### DISCUSSION TITLE
Welcome to Tafe230121 Discussions!

### DISCUSSION THREAD

**Discussion by WilLes03** [WilLes03/Tafe230121#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WilLes03** [comment]
What colour scheme should we use for the program?

**Reply by ashikstha** [reply]
we can use similar colour used in mockup given to us.

**Comment by WilLes03** [comment]
Should we format the pages with stack panels, or use margins?

**Reply by ashikstha** [reply]
I think using stackpanel is best practice

**Comment by ashikstha** [comment]
I have finished my feature part, When can we meet up next?

**Reply by WilLes03** [reply]
How about Monday 27th at 4pm Adelaide time

**Comment by ashikstha** [comment]
What should we name the feature pages?

**Reply by WilLes03** [reply]
featureMortgageCalculator and featureCurrencyCalculator

**Comment by ashikstha** [comment]
I have forked, added Trip Calculator and sent you a pull request. Can you please check that.

**Reply by WilLes03** [reply]
Yes will look at it now

**Comment by WilLes03** [comment]
I have have tested the trip calculator button and everything is working, can we meet up later today to submit?

**Reply by ashikstha** [reply]
Great, Sure we can meet at 4pm later today.
